BlockEntityRenderer/FruitpressContentsRenderer.cs

```diff
@@ -98,9 +96,10 @@ public void reloadMeshes(JuiceableProperties props, bool mustReload)
             int y;
             if (stack.Collectible.Code.Path == "rot")
             {
-                textureLocation = new AssetLocation("block/rot/rot");
-                y = GameMath.Clamp(stack.StackSize / 4, 1, 9);
-            } else
+                textureLocation = new AssetLocation("block/wood/barrel/rot");
+                y = GameMath.Clamp(stack.StackSize / 2, 1, 9);

```

Comment:
This looks better visually / 2 in my opinion.  Since the stack can never be bigger than 10 it ends up returning only 2 or 1, meaning it’s less obvious how much rot is inside comparatively, with / 2 it can range from 5 down to 1, meaning it still ends up not being full, but still having enough to visually parse how much material is inside.